shared/more/style-sheet.render.native.js

```diff
@@ -1,12 +1,240 @@
+/* @flow */
 import React, {Component} from 'react'
-import {View, Text} from 'react-native'
+import {ScrollView, Text} from 'react-native'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import Container from './dev-container.native'
+import {Box} from '../common-adapters'
+// import {Button, Input, Text, Terminal, FormWithCheckbox} from '../common-adapters'
+// import DropdownDemo from './components/dropdown.desktop'
 
 export default class Render extends Component {
   render () {
+    // const Space = () => <View style={{height: 20}}/>
+
     return (
-      <View>
-        <Text>TODO</Text>
-      </View>
-  )
+      <ScrollView>
+        <Container title='Dropdown'>
+          <Text>TODO</Text>
+        </Container>
+        <Container title='Text'>
+          <Text>TODO</Text>
+        </Container>
+        <Container title='Colors'>
+          <Box style={{...globalStyles.flexBoxColumn}}>
+            {Object.keys(globalColors).sort().map(c => {
+              return (
+                <Box key={c} style={{...globalStyles.flexBoxRow, margin: 5}}>
+                  <Box style={{width: 60, height: 60, backgroundColor: globalColors[c]}}/>
+                  <Box style={{...globalStyles.flexBoxColumn, justifyContent: 'center', marginLeft: 5}}>
+                    <Text type='Body'>{c}</Text>
+                    <Text type='Body' small>{globalColors[c]}</Text>
+                  </Box>
+                </Box>
+              ) }
+            )}
+          </Box>
+        </Container>
+        <Container title='Buttons'>
+          <Text>TODO</Text>
+        </Container>
+        <Container title='Icons'>
+          <Text>TODO</Text>
+        </Container>
+        <Container title='Inputs'>
+          <Text>TODO</Text>
+        </Container>
+      </ScrollView>
+    )
+
+    // return (
```

Comment:
I'd prefer not commiting this since it's just a commented out copy of style-sheet.desktop, right?